cmd/neblictl/internal/controlplane/client.go

```diff
@@ -129,17 +125,25 @@ func doesResourceAndSamplerMatch(resourceParameter, samplerParameter string, res
 
 }
 
-func (c *Client) getSamplers(ctx context.Context, resourceParameter string, samplerParameter string, streamUIDParameter string, cached bool) (map[resourceAndSampler]*control.Sampler, error) {
+func (c *Client) getSamplers(ctx context.Context, resourceParameter string, samplerParameter string, streamNameParameter string, cached bool) (map[resourceAndSampler]*control.Sampler, error) {
 	samplers, err := c.getAllSamplers(ctx, cached)
 
 	// Iterate over all samplers and select the ones matching the input
 	resourceAndSamplers := map[resourceAndSampler]*control.Sampler{}
 	for resourceAndSamplerEntry, samplerData := range samplers {
 		if doesResourceAndSamplerMatch(resourceParameter, samplerParameter, resourceAndSamplerEntry) {
 
-			_, ok := samplerData.Config.Streams[control.SamplerStreamUID(streamUIDParameter)]
-			if streamUIDParameter != "" && streamUIDParameter != "*" && !ok {
-				// StreamUID parameter contains a valid uid that the current sampler does not contain. Skip it
+			// Check if stream with the provided name exists
+			var ok bool
+			for _, stream := range samplerData.Config.Streams {
```

Comment:
we are using https://pkg.go.dev/slices@master#Index in several places, I think we can just use it here as well

Reply:
Just realized Streams is a map, the function cannot be replaced